tao_compiler/mlir/disc/IR/hlo_disc_ops.cc

```diff
@@ -441,6 +441,146 @@ LogicalResult SparseReshapeOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// SparseFillEmptyRowsOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult SparseFillEmptyRowsOp::reifyReturnTypeShapes(
+    OpBuilder& builder, ValueRange operands,
+    SmallVectorImpl<Value>& reifiedReturnShapes) {
+  SparseFillEmptyRowsOp::Adaptor adaptor(operands);
+  // index 0
+  auto indices_type = adaptor.indices().getType().cast<ShapedType>();
```

Comment:
just curious why is not `RankedTensorType` instead of `ShapedType`?

Reply:
changed to `RankedTensorType `